projects/flosportsinc/ng-lazy-load/src/lib/flo-lazy-load.module.ts

```diff
@@ -0,0 +1,9 @@
+import {CommonModule} from '@angular/common';
```

Comment:
you can remove these semicolons

Reply:
I just removed all the semicolons. Angular generated them, so there were several in other files breaking linting as well.

Reply:
blaming it on angular, smh